roomdb/sqlite/members.go

```diff
@@ -20,7 +20,7 @@ type Members struct {
 	db *sql.DB
 }
 
-func getAliases(mEntry *models.Member) []roomdb.Alias {
+func (m Members) getAliases(mEntry *models.Member) []roomdb.Alias {
```

Comment:
If you don't need the receiver, you can also do this:

```suggestion
// getAliases returns the JOIN-ed aliases for a member as a the wanted roomdb type.
func (Members) getAliases(mEntry *models.Member) []roomdb.Alias {
```